src/Client.php

```diff
@@ -0,0 +1,110 @@
+<?php
```

Comment:
[PSR-18](https://github.com/php-fig/http-client)?

Reply:
PSR18 is for an HTTP client but does not support asynchronicity - unfortunately, there's no standard for that yet outside of HTTPlug